cluster/cluster.go

```diff
@@ -122,6 +100,47 @@ func (c *Cluster) NewClient(serviceName string) (*Client, error) {
 	return newClient(c.localAddr, serviceName, c.Registry)
 }
 
+func memberAdd(ctx context.Context, client *clientv3.Client, cfg embed.Config) (string, error) {
+	peerUrlStrings := make([]string, len(cfg.LPUrls))
```

Comment:
is this needed because an etcd node can have more than one peerURL? will etcd treat these as the same node?

Reply:
During my testing, if theres more than one peerUrl, the initial cluster needs to be like "node1=peerUrl1,node1=peerUrl2"

Reply:
I can remove the functionality of having multiple peerUrls and just have take the first peerUrl if we want, I just added it because etcd has that funcitonality

Reply:
I think its the same embed server, it just has 2 dif peerUrls, but the initialCluster string is different in these cases

Reply:
IMO I think we should keep it as one peer URL for simplicity, but I don't feel strongly about it

Reply:
I decided to keep it, otherwise we'd have to throw errors if they try to provide multiple in the etcd config or they might be confused as to why certain peerUrls may not be used